src/main/java/pages/CoursePage.java

```diff
@@ -0,0 +1,18 @@
+package pages;
+
+import io.qameta.allure.Step;
+import org.apache.log4j.Logger;
+
+public class CoursePage extends BasePage {
+    private static final Logger logger = Logger.getLogger("Course page logger");
+
+    public CoursePage() {
+        super();
+    }
+
+    @Step("Get course page url")
+    public String getCoursePageUrl() {
```

Comment:
getURL() method can be moved to BasePage as it's common functionality for all the pages, right?